_harp/academy/_prerequisites.md

```diff
@@ -1,63 +1,62 @@
 
-[What is Code?](http://www.bloomberg.com/graphics/2015-paul-ford-what-is-code/), a recent feature by Paul Ford.
+Watch parts 2 and 3 of [Crockford on JavaScript](https://www.youtube.com/watch?v=JxAXlJEmNMg&list=PL7664379246A246CB). A wonderful series of lectures by the same Crockford as above. A panoramic overview of JavaScript at an advanced level. These two talks are hard to follow, but anything you can take from them is valuable.
 
+#### Extras
+If you have some spare time, or fancy a break from Codewars, these resources are just for fun!
 
-**Watching**	
-[Computer Science: Past, Present, and Future](http://youtu.be/5Tk09c0FQ3M), a talk by Ed Lazowska.
+**Reading**  
+[Paul Ford - What is Code?](http://www.bloomberg.com/graphics/2015-paul-ford-what-is-code/) is a great article about his impression of the tech industry and the life of a programmer. It's long, but quite light to read.
 
-[The Future Doesn't Have to Be Incremental](http://youtu.be/gTAghAJcO1o), a talk by Alan Kay.
+[Steve Yegge's Google platforms rant](https://plus.google.com/+RipRowan/posts/eVeouesvaVX) shows you how passionate someone can be about their tech company's philosophy.
 
-[Inventing on Principle](http://vimeo.com/36579366), a talk by Bret Victor.
+**Watching**  
+[Inventing on Principle](https://vimeo.com/36579366), a talk by Bret Victor.
 
-[Crockford on JavaScript](https://www.youtube.com/watch?v=JxAXlJEmNMg&list=PL7664379246A246CB). A wonderful series of lectures by Doug Crockford. A panoramic overview of JavaScript at an advanced level. Watch each lecture until you are no longer enjoying the experience and then fast forward or move onto the next one (Lecture 3 is probably the hardest to follow). You may want to rewatch these lectures at the end of the course.
-
-## UX
-
-Follow Scott Klemmer's [Human-Computer Interaction Design](https://www.coursera.org/course/hcidesign) on Coursera. (You do not have to do all the exercises, but do watch the lectures)
-
-## Startups
-
-Follow Steve Blank's [How to Build a Startup](https://www.udacity.com/course/how-to-build-a-startup--ep245) on Udacity. (about 24 hours)
+[Computer Science: Past, Present, and Future](https://www.youtube.com/watch?v=5Tk09c0FQ3M&feature=youtu.be), a talk by Ed Lazowska.
 
+[The Future Doesn't Have to Be Incremental](https://www.youtube.com/watch?v=gTAghAJcO1o&feature=youtu.be), a talk by Alan Kay.
 
+Anything with Douglas Crockford in!
 
+**Before you arrive (after course offer)**
+* [Learn Git](https://www.codecademy.com/learn/learn-git)
+* [Learn the Command Line](https://www.codecademy.com/learn/learn-the-command-line)
```

Comment:
Equally, the 'Before you arrive' could be removed as this is just to send to FACx a week / 2 weeks before they arrive